src/main/java/core/mate/academy/model/Excavator.java

```diff
@@ -1,13 +1,13 @@
 package core.mate.academy.model;
 
-/**
- * Add some custom fields that could be only in Excavator
- * Do not remove no-args constructor
- */
 public class Excavator extends Machine {
```

Comment:
The Excavator class should have private fields with getters and setters to encapsulate its properties. Direct access to class fields is discouraged.